spec/services/s3_query_service_spec.rb

```diff
@@ -18,13 +18,6 @@
     {
       is_truncated: false,
       contents: [
-        {
-          etag: "\"#{s3_etag1}\"",
-          key: s3_bucket_key,
-          last_modified: s3_last_modified1,
-          size: 0,
-          storage_class: "STANDARD"
-        },
```

Comment:
We are not seeing this on the actual AWS responses. The responses include files and directories but not the bucket_key as the first response.